compiler/integration-tests/test/browser/compile_prove_verify.test.ts

```diff
@@ -33,14 +33,11 @@ const suite = Mocha.Suite.create(mocha.suite, 'Noir end to end test');
 suite.timeout(60 * 20e3); //20mins
 
 function getCircuit(noirSource: string): CompiledProgram {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  initializeResolver((id: string) => {
-    logger.debug('source-resolver: resolving:', id);
-    return noirSource;
```

Comment:
Can create issue to remove source resolver

Reply:
Removed in https://github.com/noir-lang/noir/pull/3791